generators/base/src/GeneratorAgentClient.ts

```diff
@@ -78,7 +78,12 @@ export class GeneratorAgentClient {
             logger: this.logger
         });
         this.logger.debug(`Installing ${GENERATOR_AGENT_NPM_PACKAGE} ...`);
-        await npm(["install", "-f", "-g", GENERATOR_AGENT_NPM_PACKAGE]);
+        try {
+            await npm(["install", "-f", "-g", GENERATOR_AGENT_NPM_PACKAGE]);
```

Comment:
@amckinney i think it would be better to just not make this call if selfHosted. What do you think?